changelog.d/issue-7965

```diff
@@ -0,0 +1,4 @@
+synopsis: Ensure that v2-sdist command respects the --ignore-project flag
+packages: cabal-install
+issues: #7965
+prs: #8109
```

Comment:
I think the changelog should include the comment above:

```
-- If the "--project-file" flag is set, then this [--ignore-project] will always be false
```

As it is a property which will affect the general handling of both flags for all commands

Reply:
And i think it also should be added to docs for `--ignore-project` but i cant find any entry for that flag in docs! 🤔

Reply:
I added that line to the change log. And I added a snippet to the docs under the definition for the "--project-file" flag. I would add some help text to the "--help" flag output but I can't find anything for the "--project-file" flag as I would assume they would go together.,,

Reply:
Many thanks for take care of docs 😸